functional/ek-cert-use-ek_check_script/test.sh

```diff
@@ -51,12 +59,25 @@ _EOF"
     rlPhaseEnd
 
     rlPhaseStartTest "Add keylime agent and check genuine of TPM via ek_check_script option"
-        rlRun "keylime_tenant -v 127.0.0.1 -t 127.0.0.1 -u $AGENT_ID --allowlist allowlist.txt --exclude excludelist.txt -f excludelist.txt -c update"
-        rlRun "limeWaitForAgentStatus $AGENT_ID 'Get Quote'" 
+        rlRun -s "keylime_tenant -v 127.0.0.1 -t 127.0.0.1 -u $AGENT_ID --allowlist allowlist.txt --exclude excludelist.txt -f excludelist.txt -c update"
+        rlAssertGrep "AGENT_UUID=$AGENT_ID" $rlRun_LOG -E
+        rlAssertGrep "EK=-----BEGIN PUBLIC KEY-----" $rlRun_LOG -E
+        rlAssertGrep "EK_CERT=[^ ]+" $rlRun_LOG -E
+        rlAssertGrep "EK_TPM=[^ ]+" $rlRun_LOG -E
+        rlAssertGrep "PROVKEYS={}" $rlRun_LOG -E
```

Comment:
remove `-E` from this line.

Reply:
In newer version of PR, need use -E for extended expressions because I try to grep curly brackets.